cultivation/scripts/literature/metrics_literature.py

```
@@ -0,0 +1,166 @@
+        'self_rated_novelty_personal': Column(pa.Float, nullable=True),
+        'self_rated_comprehension': Column(pa.Int, nullable=True),
+        'self_rated_relevance': Column(pa.Int, nullable=True),
+        'flashcards_created': Column(pa.Int, nullable=True),
+        'docinsight_queries_during_session': Column(pa.Int, nullable=True),
+        'notes_length_chars_final': Column(pa.Int, nullable=True),
+        'pages_viewed_count': Column(pa.Int, nullable=True),
+        'highlights_made_count': Column(pa.Int, nullable=True)
+    }, coerce=True)
+
+
+import argparse
```

Ruff (0.11.9)

Redefinition of unused `argparse` from line 7

Remove definition: `argparse`

(F811)